### DISCUSSION TITLE
How use Graph Analytical Algorithms from Persistent Graph Store

### DISCUSSION THREAD

**Discussion by iamdrq** [alibaba/GraphScope#1932] [General]
Hi, I have a question about as title:

`
graph1 = graphscope.conn().g()
`
`
graph2 = graphscope.session().g()
`

The graph1 without  `subgraph` function, then can't direct convert to simple_graph to use built-in graph algorithms.
so, Is it load graph1 to graph2 do this by hand ? or hava a bridge do tansform between `graph1`  and  `graph2` ?

**Comment by sighingnow** [comment]
> Is it load graph1 to graph2 do this by hand ? 

Yes, currently.

> or hava a bridge do tansform between graph1 and graph2 ?

Such a feature is already in our plan and hopefully be delivered in the next few monthes.

**Reply by iamdrq** [reply]
Ok, thanks for your reply
